arm-compute/2016-03-30/swagger/containerService.json

```diff
@@ -377,7 +409,8 @@
           "properties": {
             "adminUsername": {
               "type": "string",
-              "description": "The administrator username to use for all Linux VMs"
+              "description": "The administrator username to use for all Linux VMs",
+              "pattern": "^[a-z][a-z0-9_-]*$"
```

Comment:
@amarzavery @brendandburns This means lowercase only. It was working great before with upper case, and it's even your example online:
https://msdn.microsoft.com/en-us/library/azure/mt711471.aspx
Are we sure of this pattern?